Makefile

```diff
@@ -154,7 +154,7 @@ docker-start:
 docker-stop:
 	@docker stop `cat .docker-id`
 
-eunit: export BUILDDIR = $(shell pwd)
+eunit: export BUILDDIR = $(shell pwd) ERL_FLAGS = "-config `pwd`/tmp/etc/sys.config"
```

Comment:
This wouldn't work. Actually, all this line will get assigned to BUILDDIR, so it became something like:
`"/home/kxepal/projects/asf/couchdb/couchdb ERL_FLAGS = \"-config `pwd`/tmp/etc/sys.config\""`